bodhi/client/__init__.py

```diff
@@ -337,6 +331,17 @@ def download(url, **kwargs):
                         click.echo("WARNING: download of {0} failed!".format(build['nvr']))
 
 
+def read_notes(**kwargs):
```

Comment:
This function is probably not intended to be used outside of this module, so it would be conventional to name it with a leading underscore, like ```_read_notes```.

We should also give it a docblock, explaining what it does, what kwargs is, and what it returns (if anything - I suggest having it just modify kwargs in place and not returning anything).